import-export-cli/utils/structs.go

```diff
@@ -160,6 +160,13 @@ type MigrationApisExportMetadata struct {
 	ApiListToExport []API  `yaml:"apis_to_export"`
 }
 
+type MigrationAppsExportMetadata struct {
```

Comment:
If this command is not only useful in migration scenarios, let's change the migration keyword here as well and give a more generic name; 

```suggestion
type AppsExportMetadata struct {
```

Reply:
Let's keep this as it as this will be used internally. This is to make it consistent with the other `export apis` command related changes.